partiql-test-data/partiql_test_schema.isl

```diff
@@ -0,0 +1,103 @@
+schema_header::{
+}
+
+type::{
+    name: PartiQLTestDocument,
+    type: document,
+    element: { one_of: [TestCase, Namespace] }
+}
+
+type::{
+    name: Namespace,
+    type: list,
+    element: { one_of: [TestCase, Namespace] }
+}
+
+type::{
+    name: TestCase,
+    type: { one_of: [PassParseTestCase, FailParseTestCase, FailSemanticTestCase] }
+}
+
+type::{
+    name: PassParseTestCase,
```

Comment:
I think `Syntax` tests are probably a better name than `Parse` tests. We are testing a facet of the language semantics, not the implementation. 

Thus, we have:
- syntax
- static analysis
- evaluation
- ...

Reply:
Yes, I agree. This will be changed in an upcoming `partiql-tests` PR as part of https://github.com/partiql/partiql-tests/issues/16. Also will revise the test runner parsing code to account for the name changes.